pkg/k8shandler/visualization.go

```diff
@@ -83,39 +83,40 @@ func CreateOrUpdateVisualization(cluster *logging.ClusterLogging) (err error) {
 
 func removeKibana(cluster *logging.ClusterLogging) (err error) {
 	if cluster.Spec.ManagementState == logging.ManagementStateManaged {
-		if err = utils.RemoveServiceAccount(cluster, "kibana"); err != nil {
+		namespace := cluster.Namespace
```

Comment:
if we do this in the visualization file, can we do this in the other ones too so they look consistent?

Reply:
yes